Ironmon-Tracker.lua

```diff
@@ -11,55 +11,80 @@ function Main.Initialize()
 		Main.DisplayError("This version of BizHawk is not supported for use with the Tracker.\n\nPlease update to version 2.8 or higher.")
 		return false
 	end
 
-	-- Verify the tracker files are in the same folder, as often they will accidentally get separated
-	local filecheck = io.open(Main.DataFolder .. "/Inifile.lua", "r")
-	if filecheck ~= nil then
-		io.close(filecheck)
-	else
-		print("Unable to load required tracker files.\nMake sure all of the downloaded Tracker's files are still together.")
-		Main.DisplayError("Unable to load required tracker files.\n\nMake sure all of the downloaded Tracker's files are still together.")
-		return false
+	-- Attempt to load the required tracker files
+	for _, file in ipairs(Main.TrackerFiles) do
+		local path = Main.DataFolder .. file
+		if Main.FileExists(path) then
+			dofile(path)
+		else
+			print("Unable to load " .. path .. "\nMake sure all of the downloaded Tracker's files are still together.")
+			Main.DisplayError("Unable to load " .. path .. "\n\nMake sure all of the downloaded Tracker's files are still together.")
+			return false
+		end
 	end
 
-	-- Import all scripts before starting the main loop
-	dofile(Main.DataFolder .. "/Inifile.lua")
-	dofile(Main.DataFolder .. "/Constants.lua")
-	dofile(Main.DataFolder .. "/data/PokemonData.lua")
-	dofile(Main.DataFolder .. "/data/MoveData.lua")
-	dofile(Main.DataFolder .. "/data/MiscData.lua")
-	dofile(Main.DataFolder .. "/data/RouteData.lua")
-	dofile(Main.DataFolder .. "/Memory.lua")
-	dofile(Main.DataFolder .. "/GameSettings.lua")
-	dofile(Main.DataFolder .. "/screens/InfoScreen.lua")
-	dofile(Main.DataFolder .. "/Options.lua")
-	dofile(Main.DataFolder .. "/Theme.lua")
-	dofile(Main.DataFolder .. "/ColorPicker.lua")
-	dofile(Main.DataFolder .. "/Utils.lua")
-	dofile(Main.DataFolder .. "/screens/TrackerScreen.lua")
-	dofile(Main.DataFolder .. "/screens/NavigationMenu.lua")
-	dofile(Main.DataFolder .. "/screens/SetupScreen.lua")
-	dofile(Main.DataFolder .. "/screens/GameOptionsScreen.lua")
-	dofile(Main.DataFolder .. "/screens/TrackedDataScreen.lua")
-	dofile(Main.DataFolder .. "/Input.lua")
-	dofile(Main.DataFolder .. "/Drawing.lua")
-	dofile(Main.DataFolder .. "/Program.lua")
-	dofile(Main.DataFolder .. "/Pickle.lua")
-	dofile(Main.DataFolder .. "/Tracker.lua")
-
+	print("Successfully loaded required tracker files")
 	return true
 end
 
+-- Returns true if Bizhawk version is older than 2.8
+function Main.UnsupportedBizhawkVersion()
+	-- Significantly older Bizhawk versions don't have a client.getversion function
+	if client.getversion == nil then return true end
+
+	-- Check the major and minor version numbers separately, to account for versions such as "2.10"
+	local major, minor = string.match(client.getversion(), "(%d+)%.(%d+)")
+	if major == nil or minor == nil then return true end
+
+	return (tonumber(major) < 2 or tonumber(minor) < 8)
```

Comment:
A few possibilities here that might cause this to fail:
- "Bizhawk 3" (unlikely but still possible)
- "Bizhawk 3.1" (because `minor < 8`)

And super small, minor request, but could this function be inverted to "SupportedBizhawkVersion()" and the return results flipped? I think it just reads a little nicer that way.